src/drive/web/modules/public/LightFolderView.jsx

```diff
@@ -112,7 +112,7 @@ class DumbFolderView extends React.Component {
     return (
       <Main isPublic>
         <Topbar>
-          <Breadcrumb isPublic onFolderOpen={this.props.fetchFolder} />
+          <Breadcrumb isPublic onFolderOpen={this.navigateToFolder} />
```

Comment:
Where do we still use fetchFolder?

Reply:
It's used in `loadCurrentFolder` and `navigateToFolder`, ie. on the initial mount, view reload and navigation changes. But it is not used in the jsx anymore, only the class functions.

Reply:
We can add a test to click on the breadcrumb and check if the URL has changed in the way we want, no?

Reply:
I tried to add a test like this, but it requires rendering pretty much the whole app, including a complete store and router. I don't see a way to do it in a test env in a reasonable amount of time.

I added a test that uses shallow rendering instead, that tests that calling `navigateToFolder` calls `router.push` with the correct path. It wouldn't have caught the breadcrumb problem, but we would have seen that the URL was called with `/folder/123` instead of `/files/123` prior to 3585b9b42d5486d70c15542559efde198c069a07